src/main/java/pl/javastart/task/Ticket.java

```diff
@@ -0,0 +1,65 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class Ticket {
+    public static final String TYPE_ONLINE = "online";
+    public static final String TYPE_STANDARD = "standard";
+    public static final String TYPE_GIFT = "gift";
+    private int id;
+    private String eventName;
+    private Address address;
+    private String type;
+    private int price;
+    private double discount;
+
+    private static int TICKET_COUNTER = 0;
```

Comment:
to nie jest stała, także zapis powinine być tak jak zwykłych pól: `ticketCounter`